src/main/java/techcourse/myblog/controller/CommentController.java

```diff
@@ -20,28 +21,19 @@ public CommentController(CommentService commentService) {
     }
 
     @PostMapping
-    public String createComment(CommentDto commentDto, @ModelAttribute User user) {
-        Long articleId = commentService.save(commentDto, user);
-        return "redirect:/articles/" + articleId;
+    public ResponseEntity<ResponseCommentDto> createComment(@RequestBody RequestCommentDto requestCommentDto, @ModelAttribute User user) {
+        return new ResponseEntity<>(commentService.save(requestCommentDto, user), HttpStatus.OK);
```

Comment:
```suggestion
        return ResponseEntity.ok(commentService.update(requestCommentDto, commentId, user));
```
정적팩토리 메서드를 활용해보는 것은 어떨까요